src/main/java/com/alipay/oceanbase/rpc/location/LocationUtil.java

```diff
@@ -476,6 +478,36 @@ private static void getObVersionFromRemote(Connection connection)
         }
     }
 
+    // check tenant exist or not
+    private static void CheckTenantExistFromRemote(Connection connection, TableEntryKey key)
+                                                                     throws ObTableEntryRefreshException {
+        PreparedStatement ps = null;
+        ResultSet rs = null;
+        try {
+            ps = connection.prepareStatement(OB_TENANT_EXIST_SQL);
+            ps.setString(1, key.getTenantName());
+            rs = ps.executeQuery();
+            if (rs.next()) {
+                String tenantString = rs.getString("tenant_id");
```

Comment:
What do you do after you get _tenantString_? It seems that _tenantString_ is never used again.
If there is no tenant in the server, what this will return?

Reply:
already edit